src/inference/dev_api/cpp_interfaces/interface/ie_iexecutable_network_internal.hpp

```diff
@@ -183,6 +190,12 @@ class INFERENCE_ENGINE_API_CLASS(IExecutableNetworkInternal)
      * @note Needed to correctly handle ownership between objects.
      */
     std::shared_ptr<IInferencePlugin> _plugin;
+
+    /**
+     * @brief A pointer to a plugin library.
+     * @note Needed to correctly handle ownership between objects.
+     */
+    std::shared_ptr<void> _so;
```

Comment:
who sets this for AUTO / MULTI? I see it's set in a single place in `BATCH` device

Reply:
currently only BATCH needs to set this, this is to handle this case: AUTO:GPU + batch enabled + network not batchable:
https://github.com/songbell/openvino/blob/5f3cadcfc4977665d8cf5682fcdda269d15a5651/src/plugins/auto/auto_schedule.cpp#L561